x/auth/ante/sigverify_test.go

```diff
@@ -147,15 +162,20 @@ func TestSigVerification(t *testing.T) {
 	gasLimit := testdata.NewTestGasLimit()
 
 	spkd := ante.NewSetPubKeyDecorator(suite.accountKeeper)
-	svd := ante.NewSigVerificationDecorator(suite.accountKeeper, suite.clientCtx.TxConfig.SignModeHandler())
+	anteTxConfig := authtx.NewTxConfigWithTextual(
```

Comment:
I hope it's clear with this PR that with Textual enabled, we will need 2 TxConfigs:
- one for the antehandlers (provided by the Tx module)
- one for the client TxBuilder (see L135, plugged into clientCtx)

Reply:
It is a bit confusing I think given that they both do the same thing. Except one with a bank keeper and the other using gRPC. Especially since one constructor lives in client and the other on in auth/tx.

Reply:
Agreed. I initially tried to put them in `x/tx/textual`, but unfortunately `NewTextualWithBankKeeper` needs access to `github.com/cosmos/cosmos-sdk/bank/types`.

So I put them in the tx module now: `x/auth/tx/config`.